src/templates/role.js

```diff
@@ -0,0 +1,167 @@
+import React, { useState, useEffect } from "react"
+import { graphql } from "gatsby"
+import styled from "styled-components"
+import { amplitudeLogEvent } from "utils/amplitude"
+import { Link } from "gatsby"
+
+import { SB_SALMON, SB_NAVY, SB_ORANGE } from "@colors"
+import RoleContent from "components/RolePage/RoleContent"
+import Layout from "components/layout"
+import SEO from "components/seo"
+import Section from "styles/components/Section"
+import Testimonoial from "../components/ApplyPage/testimonial"
+
+export const ROLE_COLOR_MAPPING = {
+  designer: SB_SALMON,
+  developer: SB_ORANGE,
+}
+
+const BlueFontSection = styled(Section)`
+  color: ${SB_NAVY};
+  padding-bottom: 50px;
+  min-height: 75vh;
+`
+
+const Header = styled.h1`
+  letter-spacing: 0.15em;
+  text-align: center;
+  font-weight: 600;
+  font-size: 2.5em;
+
+  @media (min-width: 1000px) {
+    font-size: 3em;
+  }
+`
+
+const Subtitle = styled.h3`
+  text-align: center;
+  font-weight: 400;
+  line-height: 1.5;
+  margin-bottom: 2em;
+
+  @media (min-width: 1000px) {
+    font-size: 1.5em;
+  }
+`
+
+const Breadcrumb = styled(Link)`
+  width: fit-content;
+  text-decoration: none;
+  border-bottom: 2px solid white;
+  color: ${SB_NAVY};
+  transition: color 0.3s, text-shadow 0.3s, border-bottom 0.3s;
+  &:hover {
+    border-bottom: 2px solid ${SB_NAVY};
+  }
+`
+const BreadcrumbSection = styled.span`
+  font-size: 1em;
+  font-weight: 400;
+  letter-spacing: 0.05em;
+  margin-top: 30px;
+  margin-bottom: 55px;
+  display: block;
+
+  @media (min-width: 1000px) {
+    font-size: 1.5em;
+    margin-bottom: 0px;
+    margin-left: 63px;
+  }
+`
+
+const SectionLine = styled.span`
+  color: ${SB_ORANGE};
+  border-top: 3px solid;
+  margin-top: 0em;
+  margin-bottom: 0em;
+  display: grid;
+  content: none;
+  margin-left: 10em;
+  margin-right: 10em;
+
+  @media (min-width: 1000px) {
+    margin-left: 17em;
+    margin-right: 17em;
+  }
+`
+
+const RolePage = ({ data, pageContext }) => {
+  const [selectedRole, setSelectedRole] = useState(pageContext.role)
+  const allRoles = pageContext.roles
+  const currentRoleData = data.allMarkdownRemark.edges.find(
+    roleData => roleData.node.frontmatter.role === selectedRole
+  ).node
+
+  useEffect(() => {
+    amplitudeLogEvent("View role", { role: selectedRole })
+  }, [selectedRole])
+
+  return (
+    <Layout page="apply">
+      <SEO title="Apply" keywords={[`application`]} />
+      <BlueFontSection>
+        <BreadcrumbSection>
+          <Breadcrumb to="/apply">apply</Breadcrumb> > {selectedRole}
+        </BreadcrumbSection>
+        <Header>{selectedRole.toUpperCase()}</Header>
+        <Subtitle>
+          Read more about Sandbox's {selectedRole} opportunities below.
+        </Subtitle>
+        <SectionLine />
+        <Testimonoial
```

Comment:
Thoughts on putting testimonial inside the RoleContent (between the description and the ideal qualities)? It might help break down the big block o' text that is down there.